apps/spruce/src/pages/task/taskTabs/buildBaronAndAnnotations/BuildBaronContent.tsx

```diff
@@ -67,7 +67,7 @@ const BuildBaronContent: React.FC<BuildBaronCoreProps> = ({
 
   return (
     <Wrapper data-cy="bb-content">
-      {loading && <Skeleton active paragraph={{ rows: 4 }} title={false} />}
+      {loading && <ParagraphSkeleton />}
```

Comment:
I think doing the previous comment will eliminate the need for this, since we pass `loading` down to this component. We can delete the `loading` prop as well

```suggestion
```